code/game/gamemodes/vampire/vampire_powers/umbrea_powers.dm

```diff
@@ -0,0 +1,178 @@
+/obj/effect/proc_holder/spell/self/cloak
+	name = "Cloak of Darkness"
+	desc = "Toggles whether you are currently cloaking yourself in darkness."
+	gain_desc = "You have gained the Cloak of Darkness ability which when toggled makes you near invisible in the shroud of darkness."
+	action_icon_state = "vampire_cloak"
+	charge_max = 2 SECONDS
+	vampire_ability = TRUE
+	panel = "Vampire"
+	school = "vampire"
+	action_background_icon_state = "bg_vampire"
+
+/obj/effect/proc_holder/spell/self/cloak/New()
+	..()
+	update_name()
+
+/obj/effect/proc_holder/spell/self/cloak/proc/update_name()
+	var/mob/living/user = loc
+	if(!ishuman(user) || !user.mind || !user.mind.vampire)
+		return
+	name = "[initial(name)] ([user.mind.vampire.iscloaking ? "Deactivate" : "Activate"])"
+
+/obj/effect/proc_holder/spell/self/cloak/cast(list/targets, mob/user = usr)
+	var/datum/vampire/V = user.mind.vampire
+	V.iscloaking = !V.iscloaking
+	update_name()
+	to_chat(user, "<span class='notice'>You will now be [V.iscloaking ? "hidden" : "seen"] in darkness.</span>")
+
+
+
+/obj/effect/proc_holder/spell/targeted/click/shadow_snare
+	name = "Shadow Snare (20)"
+	desc = "You summon a trap on the ground, when crossed it will blind the target, extinguish any lights they may have and ensares them."
```

Comment:
```suggestion
	desc = "You summon a trap on the ground. When crossed it will blind the target, extinguish any lights they may have, and ensnare them."
```